.storybook/themableAttributes.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable global-require */
+export default {
+  'bpk-animate-height': require('./../packages/bpk-animate-height').themeAttributes,
```

Comment:
Not everything has theme attributes?? This should only contain the packages that have themeAttributes.

Reply:
that's true, the thinking here is, because the storybook will load the themes on a story basis, I've added all the packages there so it'll work out of the box in the future, I could easily change it if it makes more sense